From 4def14dedd27cd1f414268832c0ea385c9d2d656 Mon Sep 17 00:00:00 2001
From: Jon Nettleton <jon@solid-run.com>
Date: Mon, 28 Mar 2022 14:50:45 +0200
Subject: [PATCH 11/11] ARM64: dts: imx8mp: Minor dts fixups

Enable both basler camera ports by default for the HummingBoard
as well as change the compat for spidev to workaround the kernel
warning splat in the logs, and fix the eeprom definition for the
carrier.

Signed-off-by: Jon Nettleton <jon@solid-run.com>
---
 .../boot/dts/freescale/imx8mp-hummingboard-pulse.dts     | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dts b/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dts
index b2983cee79f1..32e43400de96 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dts
@@ -182,6 +182,10 @@ &aud2htx {
 	status = "okay";
 };
 
+&basler_1 {
+        status = "okay";
+};
+
 &easrc {
 	fsl,asrc-rate  = <48000>;
 	status = "okay";
@@ -198,7 +202,7 @@ &ecspi2 {
 
 	spidev1: spi@0 {
 		reg = <0>;
-		compatible = "spidev";
+		compatible = "linux,spidev";
 		spi-max-frequency = <10000000>;
 	};
 };
@@ -284,7 +288,7 @@ adv7535_from_dsim: endpoint {
 	eeprom_carrier: eeprom@57 {
 		compatible = "st,24c02", "atmel,24c02";
 		reg = <0x57>;
-		pagesize = <16>;
+		pagesize = <8>;
 	};
 
 	rtc: rtc@69 {
@@ -345,7 +349,6 @@ &micfil {
 &mipi_csi_0 {
 	#address-cells = <1>;
 	#size-cells = <0>;
-	status = "okay";
 	clock-frequency = <266000000>;
 	assigned-clock-parents = <&clk IMX8MP_SYS_PLL1_266M>;
 	assigned-clock-rates = <266000000>;
-- 
2.35.1

